src/electionguard/publish.py

```diff
@@ -0,0 +1,78 @@
+from typing import List
+from jsons import set_serializer
+from os import mkdir, path
+from jsons import set_serializer, dump
+
+
+from .ballot_box import CiphertextBallot
+from .decryption_mediator import PlaintextTally
+from .election import CiphertextElectionContext, ElectionConstants, ElectionDescription
+from .key_ceremony import CoefficientValidationSet
+from .tally import CiphertextTally
+from .group import ElementModP, ElementModQ
+
+RECORD_DIR = "output" + path.sep
```

Comment:
I don't know if this is a great default value, although hopefully the caller of `publish` will specify something better.

Reply:
Fair point. Waiting to see what the team puts on this. Suggestions welcome.

Reply:
Other options considered were `artifact(s)` or `record(s)`

Reply:
yeah i think output could generally be considered a word reserved for build state in some cases.  my vote is for `results`